app/javascript/react/components/MonsterShow.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from "react";
+import MonsterTile from "./MonsterTile"
+
+
```

Comment:
extra space and I don't think this file should be here for this PR

Reply:
Removed from feature